test_data/publications/test_publications.json

```diff
@@ -147,7 +147,7 @@
   },
   {
     "publication_type": "JournalContent",
-    "publication_context_type": "Journal",
+    "publication_context_type": "UnconfirmedJournal",

```

Comment:
til info: husk at registreringer med `UnconfirmedJournal` ikke skal kunne publiseres. For å kunne publiseres skal de ha `type: "Journal"`

Reply:
Greit å vite